### DISCUSSION TITLE
Ubisoft launcher doesn't install/show up in steam library

### DISCUSSION THREAD

**Discussion by DOMMOMEISTER** [moraroy/NonSteamLaunchers-On-Steam-Deck#162] [Q&A]
I installed Ubisoft connect through this script before, and it worked. I uninstalled later, but now since recently if I wanna install it again it just does nothing for me, doesn't show up in my steam library or anywhere. I did it with steam closed, restarted everything multiple times, deleted and re-installed Proton GE but nothing helps. I really don't wanna have to factory reset, hoping for a fix here.

**Comment by moraroy** [comment]
Its been fixed
